app/components/numeric-inline-input.js

```diff
@@ -23,22 +23,26 @@ export default Ember.TextField.extend({
     return true;
   },
 
+  whichKey(e, key) {
+    return ((e.ctrlKey && key === 86) ||
+      key === 13 ||
+      key === 8 ||
+      key === 9 ||
+      key === 46 ||
+      key === 39 ||
+      (key >= 35 && key <= 37) ||
+      (key >= 48 && key <= 57) ||
+      (key >= 96 && key <= 105));
+  },
+
```

Comment:
how about this?
```
keyList = [13,8,9,39,46,]
return ((e.ctrlKey && key === 86) || keyList.includes(key) || (key >= 35 && key <= 37) ||
       (key >= 48 && key <= 57) ||
       (key >= 96 && key <= 105)); ))
```

or maybe you can include everything in an array and check if `keyList.includes(key)` like

```
 keyList  = [13,8,9,36,37,38,39,46,48,49,50,51,52,53,54,55,56,57,96,97,98,99,100,101,102,103,104,105] 
return ((e.ctrlKey && key === 86) || keyList.includes(key));
```